Warning: no DISPLAY environment variable.
--No graphics will be displayed.
Traceback (most recent call last):
  File "sequential_inference_6params.py", line 161, in main
    theta = torch.load('{}/step1/thetas.pt'.format(file_writer.folder))
  File "/home/macke/kanderer29/.conda/envs/sbi_env/lib/python3.7/site-packages/torch/serialization.py", line 594, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/macke/kanderer29/.conda/envs/sbi_env/lib/python3.7/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/macke/kanderer29/.conda/envs/sbi_env/lib/python3.7/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '/mnt/qb/work/macke/kanderer29/results/10000_sims_6_params/step1/thetas.pt'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "sequential_inference_6params.py", line 344, in <module>
    main(sys.argv[1:])
  File "sequential_inference_6params.py", line 169, in main
    num_workers=num_workers
  File "/mnt/qb/home/macke/kanderer29/sbi_for_eeg_data/code/utils/inference.py", line 70, in run_sim_theta_x
    simulator_stats, prior = prepare_for_sbi(simulation_wrapper, prior)
  File "/home/macke/kanderer29/.conda/envs/sbi_env/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 501, in prepare_for_sbi
    check_sbi_inputs(simulator, prior)
  File "/home/macke/kanderer29/.conda/envs/sbi_env/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 518, in check_sbi_inputs
    simulation = simulator(theta)
  File "/home/macke/kanderer29/.conda/envs/sbi_env/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 434, in batch_loop_simulator
    xs = list(map(simulator, theta))
  File "/home/macke/kanderer29/.conda/envs/sbi_env/lib/python3.7/site-packages/sbi/utils/user_input_checks.py", line 399, in pytorch_simulator
    return torch.as_tensor(simulator(theta), dtype=float32)
  File "/mnt/qb/home/macke/kanderer29/sbi_for_eeg_data/code/utils/simulation_wrapper.py", line 189, in simulation_wrapper_all
    net = set_network_weights(params)
  File "/mnt/qb/home/macke/kanderer29/sbi_for_eeg_data/code/utils/simulation_wrapper.py", line 304, in set_network_weights
    "L5_basket": params[2],
IndexError: list index out of range
Warning: no DISPLAY environment variable.
--No graphics will be displayed.
/var/spool/slurmd/job1132753/slurm_script: line 21: 15139 Segmentation fault      python3 investigating_summary_features.py eval_features
